nncf/quantization/advanced_parameters.py

```diff
@@ -130,6 +130,20 @@ class AdvancedSmoothQuantParameters:
     matmul: float = 0.95
 
 
+class BackupMode(Enum):
+    """
+    Specifies how to revert operations to their original precision.
+
+    :param FP32: Operations will be reverted to floating-point precision.
+    :param INT8_WEIGHTS: Certain operations (such as MatMul and Embedding) will
+        be reverted to representation with int8 weights, while all other operations
+        will revert to floating-point precision.
+    """
+
+    FP32 = "fp32"
```

Comment:
What about `FP16` precision?

Reply:
`FP32` here means "Revert to original precision", so for `FP16` IR operations will be reverted to `FP16` precision. I agree that this name is a little bit confusing

Reply:
We should rename this option then. To the `ORIGINAL` for example.

Reply:
Yes, I agree. I'll rename it.

Reply:
Done